src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.Llvm.cs

```diff
@@ -452,6 +462,7 @@ public enum TargetAbiType : byte
     public enum CorInfoLlvmEHModel
```

Comment:
do these enum values warrant a one line comment or the name is enough?

Reply:
The enum is duplicated in `llvm.h`, where there are tiny comments about the meanings. I don't think it's pointful to duplicate Jit-EE interface comment at this point in time (while the interface is still constantly evolving).